tools/license-checker/src/main.rs

```diff
@@ -115,24 +117,32 @@ struct Args {
     verbose: bool,
 }
 
+const ERROR_MESSAGE: &str = "error:";
+
+macro_rules! bright_red_and_bold_error_message {
+    () => {
+        ERROR_MESSAGE.bright_red().bold()
+    };
+}
```

Comment:
I'm having trouble seeing why this is a macro. Can it just be a function? Something like:

```rust
fn error_prefix() -> ColoredString {
    "error:".bright_red().bold()
}
```

Note that I intentionally shortened the name as well -- the function name was as long as its body!

Reply:
Replaced macro with function.